docs/src/getting-started/cli.md

```diff
@@ -34,7 +35,8 @@ miden-faucet start \
 
 | Option | Description | Default | Required |
 |--------|-------------|---------|----------|
-| `--endpoint` | Faucet endpoint | - | Yes |
+| `--backend-url` | URL to serve the Backend API | - | Yes |
+| `--frontend-url` | URL to serve the Frontend API | - | No |
```

Comment:
Should we update the naming here (and other similar places) as well? Maybe `--api-server-url` or just `--api-url`?